adobe_connect.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
 
   gem.add_dependency 'activesupport', '>= 2.3.17'
-  gem.add_dependency 'nokogiri',      '~> 1.13.3'
+  gem.add_dependency 'nokogiri',      '~> 1.13', '>= 1.13.3'
```

Comment:
Do we need both version specs? I think having `>= 1.13.3` makes the `~ 1.13` irrelevant.

Reply:
Actually, dependabot wants a more strict requirement:
https://github.com/zachpendleton/adobe_connect/security/dependabot/3
So I'm going to go with that.